yml/imsim.yaml

```diff
@@ -46,14 +46,14 @@ tables:
     description: Error in fiducial ICRS Right Ascension of centroid
     mysql:datatype: FLOAT
     fits:tunit: deg
-    ivoa:ucd: pos.eq.raErr;meta.main
+    ivoa:ucd: stat.error;pos.eq.ra;meta.main
   - name: coord_ra_dec_Cov
     "@id": "#Object.coord_ra_dec_Cov"
     datatype: float
     description: Covariance between fiducial ICRS Right Ascension and Declination of centroid
     mysql:datatype: FLOAT
     fits:tunit: deg^2
-    ivoa:ucd: pos.eq.ra_dec_Cov;meta.main
+    ivoa:ucd: stat.covariance;pos.eq.ra;pos.eq.dec;meta.main
```

Comment:
I think `pos.eq` may be preferable to `pos.eq.ra;pos.eq.dec` here.  I'm not aware of any existing art that uses both parameters' UCDs in combination like that.

Reply:
If there is any Astropy support for this, of course, we should prioritize compatibility with it.

Reply:
From my understanding, we decided to go with my initial change instead of your suggestion @gpdf .

Reply:
Yes, the feedback from the IVOA, as I mentioned verbally last week, is that they like explicitly specifying both variables, though there is no real code behind this anywhere.